baseapp/grpcrouter.go

```diff
@@ -111,13 +112,14 @@ func (qrt *GRPCQueryRouter) SetInterfaceRegistry(interfaceRegistry codectypes.In
 	)
 }
 
-// RegisterSimulateService registers the simulate service on the gRPC router.
-func (qrt *GRPCQueryRouter) RegisterSimulateService(
-	simulateFn simulate.BaseAppSimulateFn,
+// RegisterTxService registers the tx service on the gRPC router.
+func (qrt *GRPCQueryRouter) RegisterTxService(
```

Comment:
This is okay for now, but long term I'm not sure this belongs in baseapp.

Reply:
Moved to x/auth/tx.